plugins/cpp/parser/src/cppparser.cpp

```diff
@@ -711,7 +711,7 @@ void CppParser::initBuildActions()
   });
 }
 
-void CppParser::markByInclusion(model::FilePtr file_)
+void CppParser::markByInclusion(const model::FilePtr& file_)
```

Comment:
`FilePtr` is actually a `shared_ptr` and this function only uses it once to read from it to construct more `shared_ptr`s